### ISSUE TITLE
Dark Passenger is an ability with an MP cost

### ISSUE THREAD

**Issue by ufx** [ufx/GarlandTools#31]
http://www.garlandtools.org/db/#action/3633

Cost = 10 which accurately reflects its parameter growth (204 MP at Lv. 45), need to find other examples and check for common columns, as most ability-type actions have no costs.

**Comment by nakomaru** [comment]
Lv. 46 Dark Arts. 433 MP cost.
Lv. 60 Dark Arts. 1768 MP cost.
http://www.garlandtools.org/db/#action/3635

Lv. 46 Darkside. 108 MP cost.
http://www.garlandtools.org/db/#action/3628

Don't know if these are interesting to you, but there are also MP gain abilities like:

Lv. 60 Riot Blade. 442 MP gained on hit.
http://www.garlandtools.org/db/#action/15

Lv. 46 Siphon Strike. 216 MP gained on hit.
http://www.garlandtools.org/db/#action/3623

Lv. 46 Blood Weapon. 58 MP gained per physical hit (skills & auto attack).
http://www.garlandtools.org/db/#action/3625

Lv. 46 Blood Price. 86 MP gained each time damage is taken.
http://www.garlandtools.org/db/#action/3631

**Comment by ufx** [comment]
Thanks, I'll have a look at these soon to try and fix abilities with costs.

**Comment by Tylian** [comment]
Shield Oath and Sword Oath both have an MP cost of 266 (which is 10 base cost iirc) too.

**Comment by nakomaru** [comment]
Because both Oaths are Spells and not Abilities, both already display MP correctly.

http://www.garlandtools.org/db/#action/26,action/28

**Comment by Tylian** [comment]
Oh true, I should have looked at their classification rather than just assuming they shared a cooldown. I stand corrected!

**Comment by ufx** [comment]
Yes =).  The trick now is to find what distinguishes these abilities from others that have non-zero costs that aren't used.  Haven't found a good column yet.
